RecoLocalCalo/HcalRecAlgos/src/HFSimpleTimeCheck.cc

```diff
@@ -0,0 +1,133 @@
+#include <cstring>
+#include <climits>
+
+#include "RecoLocalCalo/HcalRecAlgos/interface/HFSimpleTimeCheck.h"
+#include "RecoLocalCalo/HcalRecAlgos/interface/HFRecHitAuxSetter.h"
+
+HFSimpleTimeCheck::HFSimpleTimeCheck(const std::pair<float,float> tlimits[2],
+                                     const float energyWeights[2*HFAnodeStatus::N_POSSIBLE_STATES-1][2],
```

Comment:
Yes, but passing by reference brings no advantage for the simple use
of arrays in this code, and any code change now would mean another
week-long delay for collecting signatures

On 06/17/2016 03:24 AM, David Lange wrote:
>
> In RecoLocalCalo/HcalRecAlgos/src/HFSimpleTimeCheck.cc <https://github.com/cms-sw/cmssw/pull/14838#discussion_r67475307>:
>
> > @@ -0,0 +1,133 @@
> > +#include <cstring>
> > +#include <climits>
> > +
> > +#include "RecoLocalCalo/HcalRecAlgos/interface/HFSimpleTimeCheck.h"
> > +#include "RecoLocalCalo/HcalRecAlgos/interface/HFRecHitAuxSetter.h"
> > +
> > +HFSimpleTimeCheck::HFSimpleTimeCheck(const std::pair<float,float> tlimits[2],
> > +                                     const float energyWeights[2*HFAnodeStatus::N_POSSIBLE_STATES-1][2],
>
> I believe you can pass this by reference as
> const float (&energyWeights)[2*HFAnodeStatus::N_POSSIBLE_STATES-1][2],
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/cms-sw/cmssw/pull/14838/files/94ddf556520df6348e6f7f6215e44341162a69a5#r67475307>, or mute the thread 
> <https://github.com/notifications/unsubscribe/AFS4-TUSnbjmdh2Kug4VblXfBd1uIg38ks5qMlnJgaJpZM4IyfiD>.
>